test/PullPayment.js

```diff
@@ -1,15 +1,19 @@
 var PullPaymentMock = artifacts.require("./helpers/PullPaymentMock.sol");
 
 contract('PullPayment', function(accounts) {
-
+  let ppce;
+  let tAMOUNT = 17*1e18;
```

Comment:
I agree, I think we prefer `amount`, in small caps even. By the way, why did you rename the other varable to `ppce`?